src/app.js

```diff
@@ -1 +1,42 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { numGenerator } = require('./numGenerator.js');
+const readline = require('readline');
+
+const digitsToGuess = numGenerator();
+
+const termianl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+console.clear();
+
+const ask = () => {
+  let bull = 0;
+  let cow = 0;
+
+  termianl.question('Guess a 4 digit number: ', (num) => {
```

Comment:
If my understanding is correct, then there's no validation for inputs of less or more than 4 digits, so the user can put a number of any length, and the game will not complain. While it may not ruin the logic, it sure does make the game somewhat broken.

Reply:
Let's add verification for: 
- length different than 4
- optionally - is the provided input can be coerced to Number